src/cmdline.rs

```diff
@@ -912,6 +916,32 @@ pub struct PxeNetworkUnwrapConfig {
     pub input: Option<String>,
 }
 
+#[derive(Debug, StructOpt)]
+pub struct DevShowInitrdConfig {
+    /// initrd image ("-" for stdin)
+    #[structopt(value_name = "initrd")]
+    pub input: String,
+    /// Files or globs to list
+    #[structopt(value_name = "glob")]
+    pub filter: Vec<String>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct DevExtractInitrdConfig {
+    /// Output directory
+    #[structopt(short = "C", long, value_name = "path", default_value = ".")]
+    pub directory: String,
+    /// List extracted contents
+    #[structopt(short, long)]
+    pub verbose: bool,
```

Comment:
Instead of naming this `--verbose`, should it be e.g. `--print-paths`? I can imagine it being useful for scripting. Whereas `verbose` sounds more like adding more info on stderr.

Reply:
I used `-v`/`--verbose` to match the name/behavior of the corresponding options in `tar` and `cpio`.